src/resources/postcss/tickets.pcss

```diff
@@ -1,86 +1,522 @@
-.tribe-tickets-column-attendees-link {
-	display: inline-block;
+@import "tickets-common.pcss";
+
+/**
+ * Ticket block view styles
+ *
+ * @since  0.3.0-alpha
+ */
+.tribe-common.tribe-block__tickets {
+	border: 1px solid var(--color-border-secondary);
+	border-radius: var(--border-radius-default);
+	margin: 1.5rem auto;
```

Comment:
curious why we're using `rem` here and in other places rather than `px`, it seems we're mixing `rem`, `em`, and `px` everywhere and would be good to stay consistent. for TEC VR we've stuck to px.